common/djangoapps/util/course.py

```diff
@@ -54,8 +55,15 @@ def get_link_for_about_page(course):
     elif settings.FEATURES.get('ENABLE_MKTG_SITE') and getattr(course, 'marketing_url', None):
         course_about_url = course.marketing_url
     else:
+        ## .. filter_implemented_name: CourseAboutPageURLRequested
+        ## .. filter_type: org.openedx.learning.course_about.page.url.requested.v1
+        about_base, _ = CourseAboutPageURLRequested.run_filter(
+            url=configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
+            org=course.id.org,
+        )
```

Comment:
I was reviewing this carefully again and found something concerning. This filter is called `CourseAboutPageURLRequested` but it's not modifying the course about page URL but the LMS URL. We could do two things to tackle this:
1. Modify the name in the opened-filters library to match what's currently done. However, this will limit the flexibility of modifying the course about page URL changes since I'll only be able to alter the LMS URL but not the end URL.
2. Change the placement of the filter here so it uses takes as URL the actual course about URL:
```
        course_about_url = '{about_base_url}/courses/{course_key}/about'.format(
            about_base_url=configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
            course_key=str(course.id),
        )

        ## .. filter_implemented_name: CourseAboutPageURLRequested
        ## .. filter_type: org.openedx.learning.course_about.page.url.requested.v1
        course_about_url, _ = CourseAboutPageURLRequested.run_filter(
            url=course_about_url,
            org=course.id.org,
        ) 
```
I find the 2nd approach the most flexible. What do you think?

Reply:
I made the changes here for you to review: https://github.com/openedx/edx-platform/commit/095d0deed7374a7bc4dd76f9ee7e0abc50d05e3b#diff-1b5b03eb00f8364d32308130c44df04cc4b8118caf83368b4ca3442d739697b0R89-L69

Reply:
I don't see this other comment reflected in this comment. https://github.com/openedx/edx-platform/pull/35142#discussion_r1786182388 should be `course_about_url, org = CourseAboutPageURLRequested`

Reply:
We should not apply this suggestion here as well: https://github.com/openedx/edx-platform/pull/35142/files#diff-b0db2096906850f5aa17a916c950277aa7a71350c123a8a7589e307043d1cf44R721-R724

Reply:
No, don't apply the suggestion here either.